src/modules/draw/draw-directive.js

```diff
@@ -433,7 +466,11 @@ angular.module('anol.draw')
                             scope.map.addInteraction(interaction);
                         });
 
-                        scope.activeLayer = layer;
+                        scope.activeLayer = layer
+                        // inital setup in case the active layer already contains features
+                        setContinueDrawing();
+
+                        scope.activeLayer.olLayer.getSource().on('change', setContinueDrawing);
```

Comment:
Maybe better use events `['addfeature', 'changefeature', 'removefeature']`

Reply:
these should be covered by the `change` event